README.md

```diff
@@ -2,118 +2,88 @@
+Kmesh is a high-performance and low overhead service mesh data plane based on eBPF and programmable kernel. Kmesh brings traffic management, security and monitoring to service communication without needing application code changes. It is natively sidecarless, zero intrusion and without adding any resource cost to application container.
 
 ## Why Kmesh
 
 ### Challenges of the Service Mesh Data Plane
 
-The service mesh software represented by Istio has gradually become popular and has become an important component of cloud infrastructure. However, the current service mesh still face some challenges:
+Service mesh software represented by Istio has gradually become popular and become an important component of cloud native infrastructure. However, there are still some challenges faced:
 
-- **Extra latency overhead at the proxy layer**: Single hop service access increases by [2~3ms](https://istio.io/latest/docs/ops/deployment/performance-and-scalability/#data-plane-performance), which cannot meet the SLA requirements of latency-sensitive applications. Although the community has come up with a variety of data plane solutions to this problem, the overhead introduced by agents cannot be completely reduced.
-- **High resources occupation**: The agent occupies extra CPU/MEM overhead, and the deployment density of service container decreases.
+- **Extra latency overhead at the proxy layer**: Add [2~3ms](https://istio.io/v1.19/docs/ops/deployment/performance-and-scalability/) latency, which cannot meet the SLA requirements of latency-sensitive applications. Although the community has come up with a variety of optimizations, the overhead introduced by sidecar cannot be completely reduced.
+- **High resources occupation**: Occupy 0.5 vCPU and 50 MB memory per 1000 requests per second going through the proxy, and the deployment density of service container decreases.
 
-### Kmesh：Kernel-native traffic governance
+### Kmesh Architecture
 
-Kmesh innovatively proposes to move traffic governance to the OS, and build a transparent sidecarless service mesh without passing through the proxy layer on the data path.
+Kmesh transparently intercept and forward traffic based on node local eBPF without introducing extra connection hops, both the latency and resource overhead are negligible.
 
-![image-20230927012356836](docs/pics/why-kmesh-arch-en.png)
+<div align="center">
+<img src="docs/pics/kmesh-arch.png" alt="kmesh-arch" width="800" />
+</div>
+
+The main components of Kmesh include:
+
+- Kmesh-daemon: The management component per node responsible for bpf prog management, xDS configuration subscribe, observability, and etc.
+- Ebpf orchestration: The traffic orchestration implemented based on eBPF, supports L4 load balancing, traffic encryption, monitoring and Simple L7 dynamic routing.
+- Waypoint: Responsible for advanced L7 traffic governance, can be deployed separately per namespace, per service.
```

Comment:
```suggestion
- **Waypoint**: Responsible for advanced L7 traffic governance, can be deployed separately per namespace, per service.
```

Perhaps it would be better to bold each item

Reply:
SGTM